test/test.js

```diff
@@ -1070,16 +1070,58 @@ suite('GaiaFastList >>', function() {
     });
   });
 
+  suite('unread styling >>', function() {
+    var el;
+
+    setup(function() {
+      var model = [
+        {
+          unread: true,
+          title: 'title 1',
+          body: 'body 1'
+        },
+        {
+          unread: true,
+          title: 'title 2',
+          body: 'body 2'
+        },
+        {
+          unread: false,
+          title: 'title 3',
+          body: 'body 3'
+        }
+      ];
+
+      el = createList('', 'unread=${unread}');
+      return el.setModel(model);
+    });
+
+    test('unread items should be have visible marker', function() {
+      var items = el.querySelectorAll('.gfl-item');
+      assert.equal(getComputedStyle(items[0], ':before').visibility, 'visible');
+      assert.equal(getComputedStyle(items[1], ':before').visibility, 'visible');
+      assert.equal(getComputedStyle(items[2], ':before').visibility, 'hidden');
+    });
+
+    test('all items with `unread` attr should have larger padding-start', function() {
+      var items = el.querySelectorAll('.gfl-item');
+      assert.equal(getComputedStyle(items[0]).paddingLeft, '18px');
+      assert.equal(getComputedStyle(items[1]).paddingLeft, '18px');
+      assert.equal(getComputedStyle(items[2]).paddingLeft, '18px');
```

Comment:
can we add one model entry where the `unread` key is undefined and check that the padding stays `9px`?